src/seedu/addressbook/commands/Command.java

```diff
@@ -14,13 +17,17 @@
 public class Command {
     protected AddressBook addressBook;
     protected List<? extends ReadOnlyPerson> relevantPersons;
-    private int targetIndex = -1;
+//    private int[] targetIndexes;
+    private Set<Integer> targetIndexes;
 
     /**
-     * @param targetIndex last visible listing index of the target person
+     * @param _targetIndexes last visible listing indexes of the target persons
      */
-    public Command(int targetIndex) {
-        this.setTargetIndex(targetIndex);
+    public Command(int... _targetIndexes) {
```

Comment:
This parameter name is not consistent with the rest of the codebase. i.e., there are no other cases of prefixing param name with an underscore